docs/main/03-research/01-vendors/04-trello/04-trello.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 4
+title: "Trello"
+---
+
+## UI/UX
+
+### User Interface
+
+Trello’s user interface is well-designed, clean, and user-friendly. It’s simple and visually appealing, especially in the workspace section, where Trello has effectively minimized complexity.
+
+![](photos/01-user-interface.jpg)
+
+### User Experience
+
+The login process is streamlined and intuitive, making it inviting and easy to sign up or log in. However, one concern is that Trello sometimes skips the password requirement when logging in, which raises privacy and security questions.
+  
+It would be beneficial to remove the **Recently Viewed** section from the workspace. Personally, I think navigation should be straightforward, with a single, clear path to access each board. The multiple ways to navigate to a board can feel redundant and cluttered.
+
+![](photos/02-user-experience.jpg)
+
+## Pricing
```

Comment:
turn these heading into level three headings and add a heading level 2 above these ones which reads as "Features"